juntagrico/lifecycle/sub.py

```diff
@@ -52,3 +52,8 @@ def check_sub_consistency(instance):
         raise ValidationError(
             _('HauptbezieherIn muss auch {}-BezieherIn sein').format(Config.vocabulary('subscription')),
             code='invalid')
+    if instance.parts.count() > 0 and instance.active_and_future_parts.count() == 0:
```

Comment:
Check `and instance.cancellation_date is None`. Otherwise cancelled subs will always get this error.